physionet-django/project/views.py

```diff
@@ -1956,7 +1956,7 @@ def published_project(request, project_slug, version, subdir=''):
     s3_uri = None
     try:
         if project.aws.is_private:
-            if has_signed_dua and request.user.is_authenticated:
```

Comment:
Doesn't this `has_signed_dua` need to be there for restricted and controlled projects?

Reply:
Not anymore. We're now checking project.aws.is_private to create the appropriate S3 URI.